### PR TITLE
Restructure moran redux

### PR DESCRIPTION
**Pull request by ohasselblad**
Restructure clustering functions to remove significance

CR @stuartlynn, please

### CODE REVIEW

File: src/py/crankshaft/crankshaft/clustering/moran.py [hunk 1]
```diff
@@ -11,7 +11,52 @@
...
+    Implementation building neighors with a PostGIS database and Moran's I core clusters with PySAL.
+    Andy Eschbacher
+    """
+    qvals = {"id_col": id_col,
+             "attr1": attr_name,
+             "geom_col": geom_col,
+             "subquery": subquery,
+             "num_ngbrs": num_ngbrs}
+
+    q = get_query(w_type, qvals)
```
Comment (stuartlynn) [inline src/py/crankshaft/crankshaft/clustering/moran.py]: Might be good to have a more descriptive name for get_query. How about construct_morans_query ? get_query reads like something we might have to use later on in a more general context

Comment (stuartlynn) [inline src/py/crankshaft/crankshaft/clustering/moran.py]: Or construct_neighbor_weights perhaps if thats what get_query is doing?

**Comment by stuartlynn** [comment]
@ohasselblad This is looking good to me. Just one minor comment about a function name that could be named to make it a little more obvious what its doing.

**Comment by ohasselblad** [comment]
thanks @stuartlynn :+1:

**Pull request merged**